lib/miq_automation_engine/service_models/miq_ae_service_storage_profile.rb

```diff
@@ -0,0 +1,4 @@
+module MiqAeMethodService
+  class MiqAeServiceStorageProfile < MiqAeServiceModelBase
+  end
```

Comment:
@agrare This exposes the model to automate but do you also want to expose some of the associations?

```ruby
  belongs_to :ext_management_system
  has_many :storage_profile_storages
  has_many :storages
  has_many :vms_and_templates
  has_many :disks
```